BlockchainSdk/Common/WalletManagerFactory.swift

```diff
@@ -370,6 +371,21 @@ public class WalletManagerFactory {
                 $0.networkService = TronNetworkService(isTestnet: testnet, providers: providers)
                 $0.txBuilder = TronTransactionBuilder(blockchain: blockchain)
             }
+        case .ton(testnet: let testnet):
+            return try TONWalletManager(
+                wallet: wallet,
+                networkService: .init(
+                    providers: TONNodeName.allCases.sorted().map {
```

Comment:
нужна сортировка? в плане что она правильно будет работать?

Reply:
> нужна сортировка? в плане что она правильно будет работать?

Сортировку хотелось бы оставить, задается приоритет использования апишек.

```suggestion
extension TONNodeName: Comparable {
    static func <(lhs: TONNodeName, rhs: TONNodeName) -> Bool { lhs.rawValue < rhs.rawValue }
}
```

```suggestion
enum TONNodeName: Int, CaseIterable {
```

Reply:
Посмотри например в польке. Мы уже готовые провайдеры передаем. 

```
        PolkadotWalletManager(network: network, wallet: wallet).then {
            let providers = network.urls.map { url in
                PolkadotJsonRpcProvider(url: url, configuration: networkProviderConfiguration)
            }
            $0.networkService = PolkadotNetworkService(providers: providers, network: network)
            $0.txBuilder = PolkadotTransactionBuilder(walletPublicKey: wallet.publicKey.blockchainKey, network: network)
        }
```